lib/osml/osml_restapi.ts

```diff
@@ -56,7 +56,6 @@ export interface OSMLRestApiProps {
 export class OSMLRestApi extends Construct {
   public requestAuthorizer: RequestAuthorizer;
   public restApi: RestApi;
-  public integration: Integration;
```

Comment:
Why is this removed? RestAPI needs an integration (for instance, Lambda function)

Reply:
It wasn't being used or set on the construct

Reply:
Oh, I was thinking it was coming from props.